packages/rancher-vsphere-cpi/charts/questions.yaml

```diff
@@ -1,4 +1,15 @@
 questions:
+  - variable: clusterType
+    label: Cluster Type
+    description: Select the type of the cluster where vSphere CPI will be deployed
+    type: enum
+    default: "rke"
+    options:
+      - "rke"
+      - "rke2"
+      - "other"
```

Comment:
On the UI, does this render as `rke` or `RKE`? Is there a way to change the display name of these options (e.g. should `rke2` be `RKE Government`)?

Reply:
I am not sure if it renders as rke or RKE.  Leave this question to others.
Yes, rke2 should be `RKE Government`

Reply:
It renders as "rke" and "rke2". Changing that would be a UI task.

Reply:
For future reference, we should use name of RKE2, not RKE government.